hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/TokensConfig.java

```diff
@@ -40,6 +40,8 @@ public record TokensConfig(
         @ConfigProperty(value = "nfts.maxBatchSizeMint", defaultValue = "10") @NetworkProperty int nftsMaxBatchSizeMint,
         @ConfigProperty(value = "nfts.maxAllowedMints", defaultValue = "20000000") @NetworkProperty
                 long nftsMaxAllowedMints,
+        @ConfigProperty(value = "nfts.maxAllowedAirdrops", defaultValue = "1000000") @NetworkProperty
```

Comment:
Do we need the `nfts` prefix here? I think this config is for both NFT and non-NFT tokens.

Reply:
We don't, copy paste error, removing it.